instrumentation/netty/netty-4-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/common/AbstractNettyChannelPipelineInstrumentation.java

```diff
@@ -109,4 +112,20 @@ public static void removeHandler(
       }
     }
   }
+
+  @SuppressWarnings("unused")
+  public static class RemoveFirstLastAdvice {
```

Comment:
I have some questions about the handling of `removelast`, if user call  `removelast` here, maybe the handler be removed will be ourHander? And the origin handler will be remained if user only call `removelast` once.

Reply:
Yeah, you're exactly right - I'll create a separate issue for this, since it has never worked correctly.
Thanks!